Wiki/NewToFvwm/WhyFvwm/index.md

```diff
@@ -10,27 +10,35 @@ Fvwm is a very old project and at one time was a fairly
 popular window manager, and is now a tiny project still used
 by lots of core users. What is it about fvwm that users like?
 
-+ Fvwm is light weight and fast. Fvwm is built on top of X11,
+{% capture fvwmtxt %}
+<p><strong>Fvwm is light weight and fast.</strong> - Fvwm is built on top of X11,
   and doesn't use a lot of additional libraries or widget
   sets, as such it has a very small foot print on top of
-  the already running X server, making it quite sleek.
+  the already running X server, making it quite sleek.</p>
 
-+ Fvwm is very stable, and preforms its task as a window manager
-  very well. It is very rare fvwm crashes, and most instability
-  is the application itself.
+<p><strong>Fvwm is very stable and performs its task as a window manager
+  very well.</strong> - It is very rare fvwm crashes, and most instability
```

Comment:
Same thing here -- doesn't Markdown allow for this?

Reply:
Riiight.  Then I guess the only thing to do here is to build it "locally" and push the result out to gh-pages.

Of course, "build it locally" could also mean in a mini container on Github itself which we use to publish the results -- that way, we can do whatever we like which would get rid of those restrictions.

Reply:
I have thought of the just uploading the html we build (this means we could also dig out my old syntax hilighter), but didn't think we could automate that. We should look into a setup like that, I would love to move the whole FvwmWindow from an include (which is why we have to use it in this strange way) to an actual plugin, that we can just have a start tag and end tag, and everything inbetween gets rendered using markdown.